pyk/src/pyk/ktool/kprove.py

```diff
@@ -98,6 +99,10 @@ def __init__(self, kompiled_directory, main_file_name=None, use_directory=None):
         with open(self.kompiled_directory / 'mainModule.txt', 'r') as mm:
             self.main_module = mm.read()
 
+    @staticmethod
+    def kompile(main_file: Path, *args, **kwargs) -> 'KProve':
+        return KProve(kompile(main_file, *args, **kwargs))
+
```

Comment:
Do we have a use-case for this? This adds a dependency between kprove/kompile which may not make sense down the road, and it's not very much work for the user to just call `kompile` themselves.

Reply:
OK. I thought you wanted the same interface as with `KEVM`.